src/main/java/frc/robot/RobotContainer.java

```diff
@@ -80,16 +88,13 @@ private void configureButtonBindings() {
             System.out.println("Spinning");
         }, shooterSubsystem));
 
-        launcherSolenoid
+        btnLauncherSolenoid
             .whenPressed(new InstantCommand(() -> shooterSubsystem.activatePiston(), shooterSubsystem))
             .whenInactive(new InstantCommand(() -> shooterSubsystem.lowerPiston(), shooterSubsystem));
 
-        btnIntakeOut
-            .whenHeld(new InstantCommand(() -> intakeSubsystem.spin(0.5), intakeSubsystem))
-            .whenInactive(new InstantCommand(() -> intakeSubsystem.spin(0), intakeSubsystem), true);
-        btnIntakeIn
-            .whenHeld(new InstantCommand(() -> intakeSubsystem.spin(-0.5), intakeSubsystem))
-            .whenInactive(new InstantCommand(() -> intakeSubsystem.spin(0), intakeSubsystem), true);
+        driveSubsystem.setDefaultCommand(new RunCommand(() -> driveSubsystem.drive(speed, rotation), driveSubsystem));
```

Comment:
Uh, do these variables even exist? Looks like you've duplicated 77-81 here, but with the old versions